string objName
float64[3] position
bool passedPropertyCheck
---
bool canBeAdded
